README.md

```diff
@@ -1,5 +1,5 @@
 # Pyrollbar
-[![Build Status](https://travis-ci.org/rollbar/pyrollbar.svg?branch=v0.15.0)](https://travis-ci.org/rollbar/pyrollbar)
+![Build Status](https://github.com/rollbar/pyrollbar/workflows/Pyrollbar%20CI/badge.svg?branch=v0.15.1)
```

Comment:
How about using the `latest release` GitHub feature instead of hardcoding version?

```suggestion
![Build Status](https://github.com/rollbar/pyrollbar/workflows/Pyrollbar%20CI/badge.svg?tag=latest)
```

Also, the current URL: https://github.com/rollbar/pyrollbar/workflows/Pyrollbar%20CI/badge.svg?branch=v0.15.1 states *no status*. Same for the current v0.15.2.

Reply:
Great idea. I didn't know about `tag=latest`.

No status for v0.15.1 I think makes sense because it was before the migration to Github Actions. It seems like v0.15.2 should work. The relevant commits on master look like CI ran and passed. Any ideas?

Reply:
The same result is for both v0.15.1 and v0.15.2.
Actually, _no status_ is also for the PR rollbar/rollbar.js#926 where it should be passing (I guess).

I can't say why this happens without further investigation, however I would suggest to make similar change in our other SDKs where applicable.